snewpdag/plugins/gen/GenPoint.py

```diff
@@ -82,16 +97,15 @@ def alert(self, data):
       dt = - np.dot(det.get_xyz(self.time), self.snr) / const.c # intersect
       logging.info('  dt before bias = {}'.format(dt))
       # store unbiased time in data['truth']
-      tcb = (self.ttuple[0], self.ttuple[1] + dt * g)
-      data['truth']['dets'][dname] = { 'true_t': normalize_time(tcb) }
+      tcb = time_base + dt.to(u.s).value
```

Comment:
Everything is in seconds now. The reason for the to(u.s) conversion is because of (new) line 97, since get_xyz returns something in length units, and const.c is in some length/time unit.  It's probably m/s, but in principle we're not sure.  So to make sure we get the value of the astropy Quantity in seconds, we convert to seconds and then take the value.

I'll try to to have a think about clearer variable names.

Reply:
Tried renaming the variables to make it a little easier to follow.  Hopefully it worked.